.github/workflows/build-validation.yml

```diff
@@ -22,8 +22,8 @@ on:
     branches: [ main ]
 
 env:
-  DOTNET_INSTALLER_CHANNEL: 'release/5.0.1xx'
-  DOTNET_DO_INSTALL: 'false'
+  DOTNET_INSTALLER_CHANNEL: 'release/6.0.1xx-preview4'
```

Comment:
@gewarren please review .yml changes

Reply:
I'm going to defer to @adegeo on this one.

Reply:
@shirhatti  can you explain the YML change added to the logging sample?

Reply:
@adegeo  cancel, we're good.